### DISCUSSION TITLE
RudderStack v1.7 changelog highlights

### DISCUSSION THREAD

**Discussion by gitcommitshow** [rudderlabs/rudder-server#3130] [Changelogs]
A new version `v1.7` of [rudder-server](https://github.com/rudderlabs/rudder-server) was released recently

## Highlights

These are only the highlights, so that you can understand how much relevant this new release is to you.
To read the full release notes, head over to [GitHub releases](https://github.com/rudderlabs/rudder-server/releases)

### 🚀  What's changed

- Roles-based access to Snowflake (#3031)
- Support for connecting Kafka via SSH (#3007)
- Internal improvements in Postgres warehouse operations (#2964)
- Improvement in event retry logic
- And more bug fixes and improvements

Read the full release notes for `v1.7` - https://github.com/rudderlabs/rudder-server/releases/tag/v1.7.0

Note: Keep the patch release (e.g. `x` in `1.7.x`) up to date to avoid any bugs

**Comment by cvlendistry** [comment]
I find it impossible to move to any release post 1.0 (open source) since the config syntax has changed so much. And the open source config plane is unusable. And there is no way to export a config from the hosted config plane.

**Reply by gitcommitshow** [reply]
Thanks for the feedback. RudderStack team is [not maintaining open-source config plane](https://github.com/rudderlabs/config-generator/issues/49) anymore. But using RudderStack Open Source data plane with the hosted config plane should not be hard. 
Please help me understand some more details so I can see how I can help here

1. Have you tried using workspace token to get the config instead? I find this approach to be easier to deploy and maintain. Here's how to do this - for [Kubernetes setup](https://www.rudderstack.com/docs/get-started/rudderstack-open-source/data-plane-setup/kubernetes/#rudderstack-hosted-control-plane) and for [Docker setup](https://www.rudderstack.com/docs/get-started/rudderstack-open-source/data-plane-setup/docker/#rudderstack-hosted-control-plane)
2. What is your current data plane version that you're migrating from? `v1.x` is backward compatible with most older versions. Unless your current version is way too old (e.g. 0.1)
3. Share an example of the config syntax that is causing the issue, I will look into how can we fix that
